netspresso/compressor/v2/compressor.py

```diff
@@ -0,0 +1,540 @@
+                ai_model_id=compression.input_model_id,
+                compression_method=compression.compression_method,
+                options=compression.options,
+            )
+            create_compression_response = compressor_client_v2.create_compression(
+                request_data=create_compression_request,
+                access_token=self.token_handler.tokens.access_token,
+                verify_ssl=self.token_handler.verify_ssl
+            )
+
+            for available_layers in compression.available_layers:
+                if available_layers.values:
+                    available_layers.use = True
+
+            if dataset_path and compression.compression_method == CompressionMethod.PR_NN:
+                self.upload_dataset(compression_id=compression.compression_id, dataset_path=dataset_path)
+
+            update_compression_request = RequestUpdateCompression(
+                available_layers=compression.available_layers,
+                options=compression.options,
+            )
+            update_compression_response = compressor_client_v2.compress_model(
+                compression_id=create_compression_response.data.compression_id,
+                request_data=update_compression_request,
+                access_token=self.token_handler.tokens.access_token,
+                verify_ssl=self.token_handler.verify_ssl
+            )
+
+            compression_info = update_compression_response.data
+
+            self.download_model(
+                model_id=compression_info.input_model_id,
+                local_path=default_model_path.with_suffix(extension),
+            )
+
+            compressed_model_info = self.get_model(model_id=compression_info.input_model_id)
+            available_options = self._get_available_options(compressed_model_info, default_model_path)
+
+            logger.info(f"Compress model successfully. Compressed Model ID: {compression_info.input_model_id}")
+
+            self.print_remaining_credit()
+
+            if compressed_model_info.detail.framework in [Framework.PYTORCH, Framework.ONNX]:
+                metadata.update_compressed_onnx_model_path(
+                    compressed_onnx_model_path=default_model_path.with_suffix(".onnx").as_posix()
+                )
+            metadata.update_compressed_model_path(
+                compressed_model_path=default_model_path.with_suffix(extension).as_posix()
+            )
+            metadata.update_model_info(framework=model_info.detail.framework, input_shapes=[model_info.detail.input_layer])
+            metadata.update_compression_info(
+                method=compression.compression_method,
+                options=compression.options,
+                layers=compression.available_layers,
+            )
+            metadata.update_results(model=model_info, compressed_model=compressed_model_info)
+            metadata.update_status(status=Status.COMPLETED)
+            metadata.update_available_options(available_options)
```

Comment:
P3. What do you think about calling the to() method provided by this variable(available_options)?